### ISSUE TITLE
Update tutorial to show use of the task parameters

### ISSUE THREAD

**Issue by oruebel** [rly/ndx-beadl#16]
Once #15 is complete, we should also update the tutorial notebook https://github.com/rly/ndx-beadl/blob/main/docs/tutorial_nwb_userdays_2022/beadl_example_nwb_userdays_2022.ipynb to show use the new task parameters

**Comment by oruebel** [comment]
@mavaylon1 do we have updated data files so that we can update the tutorial?
